packages/tf-inventory/src/inventory.js

```diff
@@ -74,8 +74,7 @@ async function listTfResources(stateFiles) {
  * and ec2Instances
  */
 async function listAwsResources() {
-  const ecsClusters = await aws.ecs().listClusters().promise();
-
+  const ecsClusters = await aws.ecs().listClusters();
```

Comment:
I think we need to remove the `aws.` to call the correct V3 library?

Reply:
Importing `aws` is going to import all of `aws-client/src/services.ts`. The version of `ecs` we're exporting there is `export const ecs = awsClient(AWS.ECS, '2014-11-13');`. I don't believe that's V3.

Reply:
Actually I take that part of that back. I see you did make the update to `export const ecs = awsClient(ECS, '2014-11-13');`. So the export is fine. We just need to change the import to only gran `ecs`, not all of `aws`

Reply:
I thought I was in a meeting with you Roger when you did this, when did this get reverted back to aws.ecs syntax?

Reply:
>Could you please advise what the suggested change should be to the import and to the stub for the tf-inventory files?

You're currently importing all of the aws services as 

```
const aws = require('@cumulus/aws-client/services');
```
What we're looking for are specific imports of just the libraries being used e.g.

```
const { ecs, ec2, es } = require(...);
```

Reply:
I did some digging and I see there's a reverted commit. 

https://github.com/nasa/cumulus/commit/6b6bbf9e4b1706c0b187b152b7ca55b1272dd637

I was not a part of this conversation. Is it because stubbing the individual libraries (e.g. `ecs`) was considered out of scope?